.travis.yml

```diff
@@ -26,30 +30,30 @@ git:
 env:
   global:
     - ISSUES_REPO_URI=webcompat/webcompat-tests/issues
```

Comment:
Uh weird, I had a comment and it disappeared. It was more or less this:

`- TRAVIS_NODE_VERSION="4"`

What if we picked the current LTS to be a bit more conservative? I imagine the tools we need will continue to work, but might result in fewer annoying regressions. I don't know.

Reply:
Ah yes, I made so many changes to this I probably forgot to mention this one. We were using node version 4, yes. I changed it to get the latest version always: https://github.com/webcompat/webcompat.com/pull/1978/files/8bb75f79b02a621af543bd683da7bc1be1b45b5a#diff-354f30a63fb0907d4ad57269548329e3R39